usr/lib/linuxmint/mintUpdate/kernelwindow.py

```diff
@@ -12,6 +12,7 @@
 import datetime
```

Comment:
You may as well modify the datetime usage everywhere to make it consistent... ideally though, it has nothing to do with this PR, it would be better done separately (i.e. modified everywhere in one go in a PR dedicated to doing just that).